Plugin/README.txt

```diff
@@ -22,7 +22,69 @@ NOTE: source tree requires submodules
     git submodule update --init --recursive
 
 ====================
-Building on Windows:
+Building the Dependencies:
+====================    
+    
+On Windows:
+
+Make sure to add this option when running cmake as 32 bit is often the default: -G"Visual Studio 14 2015 Win64"
+
+Building HDF5:
+
+1. follow instructions (Build Instructions 1-7) at https://portal.hdfgroup.org/display/support/Building+HDF5+with+CMake
+
+Note: make sure the binary is built in Release mode and is 64-bit
+
+2. This will generate a zip file. Unzip the file. The binaries will be located under lib/ and header files under include/
+
+3. copy the include/ directory into Plugin/external/HDF5
+
+4. copy the following library files to Plugin/external/HDF5/lib64: libhdf5.lib, libhdf5_hl.lib, libszip.lib
+
+Building ilmbase:
+
+1. follow instructions at README.cmake.txt (steps 1-2) that comes with ilmbase
+Note: When running cmake, make sure to also use the following options (taken from build.sh):
+
+-DCMAKE_BUILD_TYPE=Release \
+-DCMAKE_INSTALL_PREFIX="${builddir}" \
+-DCMAKE_PREFIX_PATH="${builddir}" \
+-DNAMESPACE_VERSIONING=OFF \
+-DBUILD_SHARED_LIBS=OFF
+
+Note: make sure to build in release mode and 64-bit (confirm when opening in visual studio)
+Note: should not need to build OpenEXR (steps 3-5 in README.cmake.txt)
+
+3. copy *.lib files from ${builddir}/lib to Plugin/external/ilmbase/lib64
+4. rename *.lib files (except for Half.lib) to *-2_2.lib
+5. copy ${builddir}/include directory to Plugin/external/ilmbase/
+
+Building Alembic:
+
+1. download and build zlib (v1.2.11): http://www.zlib.net/ (use cmake to build, no special args needed)
+2. after building zlib, copy zlibstatic.lib from {builddir}/Release/ to Plugin/external/HDF5/lib64
```

Comment:
HDF5 includes zlib I think; I wonder if we can just use the hdf5 library instead?

If not, then we should build HDF5 after zlib, and tell HDF5 about our custom zlib.

Reply:
HDF5 doesn't build zlibstatic.lib